Reply/app/src/main/java/com/materialstudies/reply/ui/email/EmailFragment.kt

```diff
@@ -62,6 +65,23 @@ class EmailFragment : Fragment() {
         savedInstanceState: Bundle?
     ): View? {
         binding = FragmentEmailBinding.inflate(inflater, container, false)
+
+        lifecycleScope.launch {
+            EmailUtils.selectedEmailState.collect { id ->
+                emailId = id
+                binding.email = EmailStore.get(id)
+            }
+        }
+
+        lifecycleScope.launch {
+            AdaptiveUtils.screenSizeState.collect { size ->
+                // Remove Navigation Icon in Large/XLarge screens
+                if (size == LARGE || size == XLARGE) {
+                    binding.toolbar.navigationIcon = null
+                }
+            }
+        }
+
```

Comment:
On L92:

Create a public method called `closeEmailDetailsPane()` in MainActivity and call that method instead of `findNavController().navigateUp()`